### PR TITLE
adjust configs of eth provider

### PR DESCRIPTION
**Pull request by huazhouwang**
## What does this implement/fix? Explain your changes.
1. 修改配置，ETH主网新增 https://api.etherscan.io，ETH 测试网新增 https://ropsten1.trezor.io

## Does this close any currently open issues?  
If it fixes a bug or resolves a feature request, be sure to link to that issue.
…

## Pull request type
 
_Put an `x` in the boxes that apply_
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Where has this been tested?
…

## Any other comments?
…

### CODE REVIEW

File: electrum_gui/common/provider/clients/eth/blockbook.py [hunk 1]
```diff
@@ -35,10 +35,9 @@ def __init__(self, url: str):
 
     def get_info(self) -> ProviderInfo:
         resp = self.restful.get("/api")
-        require(resp["blockbook"]["coin"] == "Ethereum")
 
         return ProviderInfo(
-            name="trezor",
+            name="blockbook",
```
Comment (taimanhui) [inline electrum_gui/common/provider/clients/eth/blockbook.py:40]: 这应该是之前改名的时候漏了的对吧？

File: electrum_gui/common/provider/configs/providers.json [hunk 2]
```diff
@@ -15,15 +15,26 @@
...
   ],
   "teth": [
+    {
+      "class": "BlockBook",
+      "url": "https://ropsten1.trezor.io"
+    },
     {
       "class": "Etherscan",
-      "url": "https://api-ropsten.etherscan.io/api",
+      "url": "https://api-ropsten.etherscan.io",
```
Comment (taimanhui) [inline electrum_gui/common/provider/configs/providers.json:37]: 是之前的url写错了？还是代码里有其它加上这个`/api`的操作，没注意到有后面这个。

**Review by taimanhui — APPROVED** [review]

File: electrum_gui/common/provider/clients/eth/blockbook.py (hunk 1 above)
Comment (huazhouwang) [reply electrum_gui/common/provider/clients/eth/blockbook.py:40]: 对，上次改名忘了

File: electrum_gui/common/provider/configs/providers.json (hunk 2 above)
Comment (huazhouwang) [reply electrum_gui/common/provider/configs/providers.json:37]: 因为我实现里会用 urljoin(url, "/api"), 所以这里就不需要配置 ‘/api’ 了

Comment (taimanhui) [reply electrum_gui/common/provider/configs/providers.json:37]: 嗯，是在etherscan里的`_call_action`里面那个"/api"和这里重复了，也算是之前这里写错了url（因为只需要除"/api"之外前面的部分）。

Comment (huazhouwang) [reply electrum_gui/common/provider/configs/providers.json:37]: 嗯，现在去掉

**Review by shuaiLiWang — APPROVED** [review]

**Pull request merged**